dataset_config/template/facet_fields.csv

```diff
@@ -1,11 +0,0 @@
-// List the facet fields you want. Each facet will go on one line.
```

Comment:
This might be a Github problem, but it looks like all the lines from here and the other facet_fields were deleted but the file itself still remains. Just wanted to make sure that the files themselves are deleted

Reply:
You can browse on mc/nhs branch to confirm there's not there:

https://github.com/DataBiosphere/data-explorer-indexers/tree/mc/nhs/dataset_config